pages/your_tasks/testing_code.md

```diff
@@ -154,7 +154,7 @@ The use of a testing framework becomes particularly important if your code grows
 
 #### Use test frameworks to automate testing
 
-Automated testing involves using special testing frameworks (e.g. {% tool "testthat" %} for R, {% tool "pytest" %} for Python, {% tool "junit" %} for Java) to automate and control the execution of tests (which is software separate from the software being tested)
+Automated testing involves using special testing frameworks (e.g. {% tool "testthat" %} for R, {% tool "pytest" %} for Python, {% tool "junit" %} for Java, {% tool "Test.jl" } module for Julia) to automate and control the execution of tests (which is software separate from the software being tested)
```

Comment:
```suggestion
Automated testing involves using special testing frameworks (e.g. {% tool "testthat" %} for R, {% tool "pytest" %} for Python, {% tool "junit" %} for Java, {% tool "Test.jl" %} module for Julia) to automate and control the execution of tests (which is software separate from the software being tested)
```

Reply:
If you don't close the `%`, the build will fail

Reply:
Oh, beginner's mistake - thanks @dgarijo !